constants.py

```diff
@@ -357,6 +368,56 @@
 
 kTargetName = "Target"
 
+kApriltagPositionDict = {  # this is gathered from a mapping of the field, derived with positions and ZYX order rotations converted to Quaternions in bulk in a spreadsheet
```

Comment:
Is that spreadsheet in google drive?  Just want to make sure someone else can recreate it.